catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -268,8 +268,8 @@ private void assertEqualMetacards(String reason, List<Metacard> metacardList1,
                 .map(Metacard::getTitle)
                 .collect(Collectors.toList());
 
-        assertThat(reason, idList1, containsInAnyOrder(idList2));
-        assertThat(reason, titleList1, containsInAnyOrder(titleList2));
+        assertThat(reason, idList1, containsInAnyOrder(idList2.toArray()));
+        assertThat(reason, titleList1, containsInAnyOrder(titleList2.toArray()));
```

Comment:
`containsInAnyOrder` wants a few different things. A collection of matchers, a vararg of matchers, or a vararg of objects. This `toArray` calls is the easiest solution; otherwise, it'd require mapping this collection to one of matchers.